src/dev/code_coverage/ingest_coverage/team_assignment/parse_owners.js

```diff
@@ -8,16 +8,15 @@
 
 import { fromEvent } from 'rxjs';
 import { map, filter, takeUntil } from 'rxjs/operators';
-import { lineRead, pathAndTeams, empties, comments, dropCCDelim } from './parse_owners_helpers';
+import { lineRead, pathAndTeams, dropCCDelim } from './parse_owners_helpers';
 import { pipe } from '../utils';
 
-const cleanAndParse = pipe(dropCCDelim, pathAndTeams);
-
 const allLines$ = (lineReader) =>
   fromEvent(lineReader, 'line').pipe(
-    filter(empties),
-    filter(comments),
-    map(cleanAndParse),
+    filter(function dropEmptiesAndDropComments(x) {
+      return x !== '' && !/^#\s{1,3}/.test(x);
```

Comment:
@wayneseymour could you share more about what do we filter here?

Reply:
Sure, its just the same as before but it's all in one for less operations. 
As a predicate function if it's passed an item that is not empty and is not a comment (preceded by a # character and 1-3 whitespace characters, the predicate function, allows that item to move to the next operation.